packages/sources/icap/tsconfig.json

```diff
@@ -5,6 +5,5 @@
     "rootDir": "src"
   },
   "include": ["src/**/*", "src/**/*.json"],
-  "exclude": ["dist", "**/*.spec.ts", "**/*.test.ts"],
-  "references": [{ "path": "../tp" }]
```

Comment:
How come we don't need this reference anymore? I believe the `icap` EA still depends on `tp` for the endpoint: https://github.com/smartcontractkit/external-adapters-js/blob/main/packages/sources/icap/src/endpoint/price.ts#L1

Reply:
Good spot. I wasn't sure if that would be the case or not, as it was importing from the tp-adapter package directly, not the tp folder. It might either be a quirk of how the references option in tsconfig works, or perhaps the github check is a bit too overly strict?

https://www.typescriptlang.org/docs/handbook/project-references.html

Currently unsure.

Either way, I am happy to re-add the reference for now.